src/ctia/http/routes/indicator.clj

```diff
@@ -35,26 +35,44 @@
     :tags ["Indicator"]
     (GET "/:id/judgements" []
       :return [StoredJudgement]
-      :path-params [id :- Long]
+      :path-params [id :- s/Str]
+      :query [params SightingsByIndicatorQueryParams]
       :summary "Gets all Judgements associated with the Indicator"
-      (not-found))
+      (if-let [indicator (read-indicator @indicator-store id)]
+        (if-let [judgements (list-judgements-by-indicators @judgement-store [indicator] params)]
```

Comment:
I'm thinking about this style of list method.  In #288, there is a new ID protocol and a CtiaID record.  What do you think about passing in a set of CtiaID instances instead of the list of records?  The problem is that these routes receive the short-ID (as in :id here), but the references need to become URLs.  The ID protocol has methods for getting either the (.short_id ...) or the (.long_id ...), so the list methods can use whatever it needs.  Also, I am thinking that I would like to avoid parsing the IDs in the store method implementations.  Instead, we can do it once here and then pass the CtiaID records down.

Still... I'm not sure.  Let me know what you think.